src/storefront/public/locales/en/product.json

```diff
@@ -2,5 +2,8 @@
   "varinats": "Variants",
   "recommended-products-title": "Recommended products",
   "recommended-products-description": "See other products that you might like",
-  "reviews": "Cusotmer reviews"
+  "reviews": "Cusotmer reviews",
```

Comment:
Typo? Or is it in loving memory of `ccommon`? 🤔

Reply:
![ThatsATypoHomerSimpsonGIF](https://github.com/ecoseller/ecoseller/assets/30175078/028c98d4-0674-43bb-8382-413b8d58a313)

Reply:
fixed